packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -118,6 +118,13 @@ export default declare((api, opts) => {
 
       Function({ node }) {
         if (skipStrip) return;
+        if (
+          node.params[0] &&
```

Comment:
```suggestion
          node.params.length > 0
```
Out-of-bound reading is slow, please avoid it: https://v8.dev/blog/elements-kinds#avoid-reading-beyond-the-length-of-the-array